kube/pod.go

```diff
@@ -450,8 +459,36 @@ func getSecurityContext(role *model.Role) helm.Node {
 		capabilities = append(capabilities, cap)
 	}
 	if len(capabilities) == 0 {
+		if createHelmChart {
+			// Conditional capabilities, pass the entire operator-specified set
+			cla := helm.NewMapping("add", helm.NewNode(fmt.Sprintf("{{ %s }}", config)))
+			cla.Set(helm.Block(notAll))
+
+			// Complete the context, with conditional privileged mode
+			sc := helm.NewMapping()
+			sc.Add("privileged", helm.NewNode(true, helm.Block(hasAll)))
+			sc.Add("capabilities", cla)
+			return sc
+		}
 		return nil
 	}
+	if createHelmChart {
+		// Conditional capabilities, fixed set, and range over
+		// operator-specified dynamic set.
+		caplist := helm.NewList()
+		for _, cap := range capabilities {
+			caplist.Add(cap)
+		}
+		caplist.Add(helm.NewNode("{{ . }}", helm.Block(fmt.Sprintf("range %s", config))))
```

Comment:
Can we `|upper` this, so that it isn't case-sensitive for the operator?

Reply:
Ah. Certainly.

Reply:
Done with commit 6d720721499381c95c34e4e4c9960da8bb23e60a